src/resources/constants.rs

```diff
@@ -1,12 +1,2 @@
 pub const TILE_SIZE: f32 = 32.0;
```

Comment:
@MatthewGriffin refactored this dumb idea where it was only being used in the concept's concerns